data-reporting-service/person-service/src/main/resources/application.yaml

```diff
@@ -11,16 +11,16 @@ spring:
           topic-name: nbs_default
       output:
         providerElastic:
-          topic-name: ${PROVIDER_ELASTIC:nrt_provider_elastic}
+          topic-name: ${PROVIDER_ELASTIC:elastic_nrt_provider}
```

Comment:
I also made the same change in my deployment branch. Will have to resolve the merge conflicts between us :)

Reply:
No problem! I am testing something that might need further changes in the model. We can discuss the convention offline.